bids/tests/data/ds005/models/ds-005_type-test_model.json

```diff
@@ -62,7 +62,6 @@
             "Name": "by-group",
             "Level": "Dataset",
             "GroupBy": [
-                "sex",
```

Comment:
Does this not work?

Reply:
Actually, no. And in previous tests this was being ignore because it was over-written in the smoke test by manually setting `GroupBy` to only `contrast`

Reply:
I'll open another issue for this